repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -57,6 +57,9 @@ stage("Performance Test") {
     // builds the Repose artifacts for performance test jobs to use
     build("repose-gradle-package")
 
+    // builds the performance test library used by all these jobs
+    build("performance-test-library")
```

Comment:
Just want to make a note that to get this to work, a build step will need to be added to the `performance-test` job to copy this artifact.